gradle.properties

```diff
@@ -15,7 +15,7 @@ pluginUntilBuild = 243.*
 pluginVerifierIdeVersions = IC-2024.1.7, IC-2024.2.4, IC-2024.3
 
 platformType = IC
-platformVersion = 2024.1
+platformVersion = 2024.3
```

Comment:
This should stay with 2024.1 - this is the lowest supported version, that this plugin is built against. This plugin should still work with 2024.1, as configured by `pluginSinceBuild`.

The properties, that control, whether this plugin is available in the marketplace for installing, is `pluginSinceBuild` and `pluginUntilBuild`.

The compatibility check is done via the gradle task [runPluginVerifier](https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-runpluginverifier) - the versions, that are tested are configured with `pluginVerifierIdeVersions` - and this detects any deprecated/incompatible usages providing a nice report.

Reply:
Right, I used it to test on 2024.3, should be put back to 2024.1. The ([snapshot version](https://github.com/jborgers/PMD-Intellij/releases/tag/v2.0.3-snapshot)) created with it does work on both IntelliJ version.